### ISSUE TITLE
Need for Validation Set?

### ISSUE THREAD

**Issue by jlevy44** [alimirzaei/TSFS#4]
Is there a need for a validation set for using TSFS? Or is the goal to optimize on just the training set? Or is the choice up to us?

**Comment by alimirzaei** [comment]
As described in the paper:
Because some of the employed datasets are not splitted to train and test subsets, we use 5-fold cross-validation on original datasets and report the mean accuracy of the five test parts.
